src/style.scss

```diff
@@ -16,7 +16,7 @@
     --plotweaver-color-grey-4: #444;
     --plotweaver-color-grey-5: #555;
     --plotweaver-color-white: white;
-    --plotweaver-color-off-white: #f0f0e0;
+    --plotweaver-color-off-white: #fffff0;
```

Comment:
This should not be changed as it is the colour used by Foundry itself for showing settings icons and such and we don't want mismatches

Reply:
Good catch, it has been reverted.